packages/linter/src/helper.ts

```diff
@@ -73,7 +74,7 @@ export function fetchToCurl(
   const headers = init.headers || {};
 
   const h = Object.keys(headers)
-    .map((k) => `-H '${k}: ${headers[k]}'`)
```

Comment:
it looks like you removed a few `()` around single arguments. Would it make sense to add `arrowParens: "avoid"` to  the prettier [config](https://github.com/ampproject/amp-toolbox/blob/main/prettier.config.js)?